tests/fsharpqa/Source/Globalization/Hindi.fs

```diff
@@ -14,8 +14,8 @@ module पिछले =
   
   // DU
   type ख़तरxनाक = 
-               | Uअलगाववादी     // There's no uppercase/lowercase in Hindi, so I'm adding a latin char
-               | Aमिलती of ख़तरनाक
+               | अलगाववादी     // There's no uppercase/lowercase in Hindi, so I'm adding a latin char
```

Comment:
Comment is out of date

Reply:
Nice .... that's the problem with comments ... they bit-rot so fast.

Reply:
fixed

Reply:
> they bit-rot so fast.

Is MSFT working on NLP versus code analysis already to help with this?

Reply:
Nope

Reply:
When I was working on Indigo the rule was no comments.   That still bothers everyone I work with :-)

Reply:
'tis a silly rule